tests/integration/features/istio-jwt/istio_jwt_scopes.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Exposing two endpoints with Istio JWT authorization strategy with scopes
+
+  Scenario: IstioJWTScopes: Calling an endpoint secured with JWT with a valid token
+    Given IstioJWTScopes: There is a deployment secured with JWT on path "/ip"
```

Comment:
More information for the given endpoints might it easier to understand the test without reading implementation, e.g. `There is one deployment that requires scope claims "read", ... and there is a deployment that requires scope claims "test", ...`
We can then also adapt the assertion text adding more info, e.g. `Calling the "/ip" endpoint with a valid "JWT" token and scope claims "read", "test" and "write"  should result in status between 200 and 299`.
Maybe it makes sense to have two scenarios one for happy and one for unhappy tests.